package.json

```diff
@@ -14,7 +14,7 @@
     "hubot"
   ],
   "dependencies": {
-    "hubot": ">= 2.6.0 < 3"
+    "hubot": ">=2.6.0 <10 || 0.0.0-development"
```

Comment:
or move `^3.0.0` to `devDependencies` and put the range above into `peerDependencies`